### PR TITLE
fix template linter errors

### PR DESCRIPTION
**Pull request by stopfstedt**

### CODE REVIEW

File: app/components/user-menu.hbs [hunk 1]
```diff
@@ -24,20 +24,20 @@
     <ClickOutside @onClickOutside={{fn (set this.isOpen) false}}>
       <ul class="menu">
         <li tabindex="-1" data-test-item>
-          <LinkTo @route="myprofile" @invokeAction={{set this.isOpen false}}>
+          <LinkTo @route="myprofile" {{on "click" (set this.isOpen false)}}>
```
Comment (jrjohnson) [inline app/components/user-menu.hbs:27]: Unfortunately this doesn't work see https://stackoverflow.com/questions/69021116/add-extra-actions-to-linkto-in-octane I have an idea for a component to solve this problem, but this should be reverted here.

File: app/components/user-menu.hbs [hunk 2]
```diff
@@ -24,20 +24,20 @@
...
             {{t "general.myProfile"}}
           </LinkTo>
         </li>
         <li tabindex="-1" data-test-item>
           <LinkTo
             @route="mymaterials"
-            @invokeAction={{set this.isOpen false}}
+            {{on "click" (set this.isOpen false)}}
```
Comment (jrjohnson) [inline app/components/user-menu.hbs:34]: see above, revert.

File: app/components/user-menu.hbs [hunk 3]
```diff
@@ -24,20 +24,20 @@
...
           >
             {{t "general.myMaterials"}}
           </LinkTo>
         </li>
         <li tabindex="-1" data-test-item>
-          <LinkTo @route="logout" @invokeAction={{set this.isOpen false}}>
+          <LinkTo @route="logout" {{on "click" (set this.isOpen false)}}>
```
Comment (jrjohnson) [inline app/components/user-menu.hbs:40]: see above, revert.

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: app/components/user-menu.hbs (hunk 3 above)
Comment (stopfstedt) [reply app/components/user-menu.hbs]: :heavy_check_mark:

File: app/components/user-menu.hbs (hunk 2 above)
Comment (stopfstedt) [reply app/components/user-menu.hbs]: :heavy_check_mark:

File: app/components/user-menu.hbs (hunk 1 above)
Comment (stopfstedt) [reply app/components/user-menu.hbs]: :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**